woodwork/table_accessor.py

```diff
@@ -673,21 +685,30 @@ def drop(self, columns):
         not_present = [col for col in columns if col not in self._dataframe.columns]
         if not_present:
             raise ColumnNotPresentError(not_present)
+        return self._get_subset_df_with_schema([col for col in self._dataframe.columns if col not in columns], inplace=inplace)
 
-        return self._get_subset_df_with_schema([col for col in self._dataframe.columns if col not in columns])
-
-    def rename(self, columns):
+    def rename(self, columns, inplace=False):
         """Renames columns in a DataFrame, maintaining Woodwork typing information.
 
         Args:
             columns (dict[str -> str]): A dictionary mapping current column names to new column names.
+            inplace (bool): If False, return a copy. Otherwise, do operation inplace and return None.
```

Comment:
Similar to above, we should mention that `inplace` is only supported when using pandas DataFrames.

Reply:
done!

Reply:
Update docstring to mention limitations